pages/canvas/canvas_activities_page.rb

```diff
@@ -376,6 +377,135 @@ def click_e_grades_export_button
       wait_for_load_and_click e_grades_export_link_element
     end
 
+    # Given a user and its Gradebook total score, returns the user's UID, SIS ID, and expected grade (based on the default grading scheme)
+    # @param user [User]
+    # @param score [Float]
+    # @return [Hash]
+    def student_and_grade(user, score)
+      grade = case score
+                when 99.9..100
+                  'A+'
+                when 95..99.89
+                  'A'
+                when 90..94.99
+                  'A-'
+                when 87..89.99
+                  'B+'
+                when 83..86.99
+                  'B'
+                when 80..82.99
+                  'B-'
+                when 77..79.99
+                  'C+'
+                when 73..76.99
+                  'C'
+                when 70..72.99
+                  'C-'
+                when 67..69.99
+                  'D+'
+                when 63..66.99
+                  'D'
+                when 60..62.99
+                  'D-'
+                when 0..59.99
+                  'F'
+                else
+                  logger.error "Invalid score '#{score}'"
+              end
+      {uid: user.uid, canvas_id: user.canvas_id sis_id: user.sis_id, score: score, grade: grade}
```

Comment:
@pfarestveit missing comma after `canvas_id: user.canvas_id`?

Reply:
Thanks for catching @johncrossman !